### PR TITLE
Keybase Shell option for Windows

### PR DESCRIPTION
**Pull request by gabriel**
@keybase/react-hackers @zanderz

### CODE REVIEW

**Comment by zanderz** [comment]
LGTM :thumbsup:

File: shared/menubar/index.js [hunk 1]
```diff
@@ -139,6 +140,17 @@ class Menubar extends Component<void, Props, void> {
     this._closeMenubar()
   }
 
+  _openShell () {
+    if (process.platform === 'win32') {
+      let shellCmd = 'start "Keybase Shell" "%APPDATA%\\Microsoft\\Windows\\Start Menu\\Programs\\Keybase\\Keybase Shell.lnk"'
```
Comment (MarcoPolo) [inline shared/menubar/index.js]: @zanderz Is this the preferred way of doing it? what if they remove keybase shell form the start menu?

**Comment by MarcoPolo** [comment]
👍 % q for @zanderz

**Comment by chrisnojima** [comment]
I don't think we necessarily need to cover this case

File: shared/menubar/index.js (hunk 1 above)
Comment (zanderz) [inline shared/menubar/index.js]: Yeah it wouldn't work if they did that, but that seems OK to me.

Comment (gabriel) [inline shared/menubar/index.js]: We should make it point to shortcut in app directory, but we need to have installer copy it there first? https://keybase.atlassian.net/browse/DESKTOP-1681

**Pull request merged**